.cirrus.yml

```diff
@@ -48,7 +48,7 @@ plugin_qa_task:
     memory: 8G
   env:
     matrix:
-      - SQ_VERSION: LATEST_RELEASE[9.9]
+      - SQ_VERSION: LATEST_RELEASE
```

Comment:
Seems unrelated?

Reply:
Sorry, I forgot to add a note to the PR. Currently master is red, we need to update ITs SQ_VERSION to make it green. Two options:
* `Rebase and merge` this PR without squashing and keep the commit in this PR
* take the commit updating `SQ_VERSION` out of this PR
Given the repository I opted for the first option, if you are strongly against that I can split it in two.

Reply:
Rebase and merge is good, I was just wondering if it was an intended change.